package.json

```diff
@@ -12,7 +12,7 @@
     "html-entities": "1.2.1",
     "prop-types": "15.5.10",
     "react": "16.0.0-alpha.12",
-    "react-native": "0.47.2",
+    "react-native": "0.55.4",
```

Comment:
that's quite a big change. Did you check if everything works well afterwards?

Reply:
I realize it is quite a big change, I wouldn't have done it if I didn't get that error I was talking about before. I did try it on my phone, and it worked as normal.